### ISSUE TITLE
AD410 Human detection on/off switch

### ISSUE THREAD

**Issue by frustreermeneer** [rroller/dahua#143]
Thanks for you work on the integration so far.  It works flawlessly here.

If possible, it would be nice to have a switch in HA to turn human detection on and off.
I hope it wasn't asked before.

**Comment by rroller** [comment]
This should be doable! I'll post back once it is done.

**Comment by rroller** [comment]
OK this isn't as easy as I thought it would be because the AD410 human detection isn't enabled/disabled the same way as a Dahua camera (which I've already added support for). I'm still poking around to see what I can find, but at this point I don't think I can add the feature because I don't know the API to call to enable/disable SmartHumanDetection

**Comment by rroller** [comment]
Looks like it is controlled via the `VideoAnalyseRule`. I'll try poking around with this...

```
VideoAnalyseRule[0][0].Enable=false
VideoAnalyseRule[0][0].Enable=true
```

**Comment by rroller** [comment]
OK this works...

On
```
http://192.168.1.162/cgi-bin/configManager.cgi?action=setConfig&VideoAnalyseRule[0][0].Enable=true
```

Off
```
http://192.168.1.162/cgi-bin/configManager.cgi?action=setConfig&VideoAnalyseRule[0][0].Enable=true
```

**Comment by rroller** [comment]
And the status can be fetched with...

```
http://192.168.1.162/cgi-bin/configManager.cgi?action=getConfig&name=VideoAnalyseRule[0][0].Enable
```

**Comment by rroller** [comment]
Got this working and released in https://github.com/rroller/dahua/releases/tag/0.9.33 

It looks like this...

<img width="312" alt="Screen Shot 2022-01-17 at 11 37 38 AM" src="https://user-images.githubusercontent.com/445655/149828444-a8997740-2c90-4775-8497-26aac90192b0.png">

**Comment by frustreermeneer** [comment]
Thanks, just tested it and it works great!
